tests/config.c

```diff
@@ -249,18 +243,7 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
          config_set_mib("cache-capacity", cfg, cache_capacity) {}
          config_set_gib("cache-capacity", cfg, cache_capacity) {}
          config_set_string("cache-debug-log", cfg, cache_logfile) {}
-         config_has_option("perform-bg-tasks")
-         {
-            for (uint8 cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
-               cfg[cfg_idx].perform_bg_tasks = TRUE;
-            }
-         }
-         config_has_option("no-perform-bg-tasks")
-         {
-            for (uint8 cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
-               cfg[cfg_idx].perform_bg_tasks = FALSE;
-            }
-         }
+         config_set_uint64("queue-scale-percent", cfg, queue_scale_percent) {}
```

Comment:
I don't see any validation checks further below to ensure that bogus values do not creep through.

I think, it will be useful to have some checks in the validation checks block from L326 to check for unsupportable or bogus values.

Reply:
Any value is allowed.